OpenRA.Mods.Common/Traits/AmmoPool.cs

```diff
@@ -45,96 +48,119 @@ public class AmmoPoolInfo : ITraitInfo
 		[Desc("Time to reload per ReloadCount on airfield etc.")]
```

Comment:
You should add `// HACK: Temporarily needed until Rearm activity is gone for good` here as well, for good measure.

Reply:
Done.